docs/concepts/Security-Best-Practices.md

```diff
@@ -0,0 +1,166 @@
+
+One example of a software supply chain attack occurs when malicious code is purposefully added to a dependency, using the supply chain of that dependency to distribute the code to its victims. Supply chain attacks are real. There are many methods to attack a supply chain, from directly inserting malicious code as a new contributor, to taking over a contributor’s account without others noticing, or even compromising a signing key to distribute software that is not officially part of the dependency.
+
+Software supply chain attacks are in and of itself rarely the end goal, rather it is the beginning of an opportunity for an attacker to insert malware or provide a backdoor for future access.
+
+![The State of the Octoverse 2020 - Vulnerability Lifecycle](media/vulnerability-lifecycle.png)
+
+## Unpatched Software
+
+The use of open source today is significant and is not expected to slow down anytime soon. Given that we are not going to stop using open-source software, the threat to supply chain security is unpatched software. Knowing that, how can you address the risk that a dependency of your project has a vulnerability?
+
+- **Knowing what is in your environment.** This requires discovering your dependencies and any transitive dependencies to understand the risks of those dependencies such as vulnerabilities or licensing restrictions.
+- **Manage your dependencies.** When a new security vulnerability is discovered, you must determine whether you are impacted, and if so, update to the latest version and security patch available. This is especially important to review changes that introduce new dependencies or regularly auditing older dependencies.
+- **Monitor your supply chain.** This is by auditing the controls you have in place to manage your dependencies. This will help you enforce more restrictive conditions to be met for your dependencies.
+
+![The State of the Octoverse 2020 - Advisories](media/advisories.png)
+
+We will cover various tools & techniques that NuGet & GitHub provides that you can use today to address potential risks inside your project. 
+
+## Knowing what is in your environment
+
+### NuGet Dependency Graph
+
+You can view your NuGet dependencies in your project by looking directly at the respective project file.
+
+This is typically found in one of two places:
+
+-	[`packages.config`](https://docs.microsoft.com/nuget/reference/packages-config) – Located in the project root.
+-	[`<PackageReference>`](https://docs.microsoft.com/nuget/consume-packages/package-references-in-project-files) – Located in the project file. 
+
+Depending on what method you use to manage your NuGet dependencies, you can also use Visual Studio to view your dependencies directly in the [Solution Explorer](https://docs.microsoft.com/visualstudio/ide/solutions-and-projects-in-visual-studio?view=vs-2019#solution-explorer) or [NuGet Package Manager](https://docs.microsoft.com/nuget/consume-packages/install-use-packages-visual-studio).
```

Comment:
```suggestion
Depending on what method you use to manage your NuGet dependencies, you can also use Visual Studio to view your dependencies directly in [Solution Explorer](https://docs.microsoft.com/visualstudio/ide/solutions-and-projects-in-visual-studio?view=vs-2019#solution-explorer) or [NuGet Package Manager](https://docs.microsoft.com/nuget/consume-packages/install-use-packages-visual-studio).
```

Reply:
Guidelines are to use relative links for docs in the same repo (.md required in the URL path), and top-level links are preferred for docs in another repo (/visualstudio/...). For the latter, no .md suffix is necessary.

Reply:
One reason is to ensure that the locale is not present in the URL.

Reply:
I usually remove the locale after warnings but will definitely keep this in mind!